pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -124,6 +132,7 @@ func (i *CommandInitOption) Validate(parentCommand string) error {
 	return nil
 }
 
+// nolint
```

Comment:
Which lint failed? complex?

Reply:
Yes we have four ways to resolve it
1、use // nolint to mark this function
2、use // nolint:gocyclo,revive to mark this function more accurately
3、writing the exclude-rule in `.golangci.yml`
4、decrease the function complexity (Seems not a better)